tree/tree/src/TTreeCache.cxx

```diff
@@ -240,8 +251,22 @@ of effective system reads for a given file with a code like
 #include "TLeaf.h"
 #include "TFriendElement.h"
 #include "TFile.h"
+#include "TMath.h"
 #include <limits.h>
 
+// TODO: Copied from TBranch.cxx
```

Comment:
This is now in 3 files (also in TBranchElement).  We need to move it to a single location (maybe RConfig.h)